hathor/transaction/storage/rocksdb_storage.py

```diff
@@ -153,6 +153,21 @@ def get_count_tx_blocks(self) -> int:
         keys_count = int(keys_bcount)
         return keys_count
 
+    def get_sst_files_sizes_by_cf(self, cfs: Optional[List[Any]] = None) -> Dict[bytes, float]:
+        """Get the SST files sizes of each Column Family in bytes
+
+        :param cfs: The list of column families, defaults to None, in which case all of them are returned
+        :return: A dict containing the names of the cfs and their sizes
+        """
+        column_families = cfs or self._db.column_families
```

Comment:
There's a slight edge case here where passing `cfs=[]` will return all column families instead of no column families. I think it should be `self._db.column_families if cfs is None else cfs` instead.

Reply:
Fixed in https://github.com/HathorNetwork/hathor-core/pull/485/commits/bddd8196b7cdf1124cc30af11d8745822c19d47c